doc/api/n-api.md

```diff
@@ -2637,6 +2637,41 @@ It is the equivalent of doing `new Object()` in JavaScript.
 The JavaScript `Object` type is described in [Section object type][] of the
 ECMAScript Language Specification.
 
+#### `napi_create_object_with_properties`
+
+<!-- YAML
+added: REPLACEME
+napiVersion: 8
+-->
+
+```c
+napi_status napi_create_object_with_properties(napi_env env,
+                                               napi_value prototype_or_null,
+                                               const napi_value* property_names,
+                                               const napi_value* property_values,
+                                               size_t property_count,
+                                               napi_value* result)
+```
+
+* `[in] env`: The environment that the API is invoked under.
+* `[in] prototype_or_null`: The prototype object for the new object, or `null`
+  for no prototype.
```

Comment:
This should document whether it refers to the JS `null` value (pretty sure it does) or the C `nullptr` value. (Converting `nullptr` to `null` here might be a nice usability improvement, though.)

Reply:
Hey @addaleax 
I did this conversion, but I was in doubt about how to properly test it because the part that lives these test_object` tests is in C format. Would it be best to create a new file just to deal with these new tests in cpp or is it easy to change the format of the file that already exists?